sdk/core/core-http/lib/coreHttp.ts

```diff
@@ -50,4 +50,5 @@ export { BasicAuthenticationCredentials } from "./credentials/basicAuthenticatio
 export { ApiKeyCredentials, ApiKeyCredentialOptions } from "./credentials/apiKeyCredentials";
 export { ServiceClientCredentials } from "./credentials/serviceClientCredentials";
 export { TopicCredentials } from "./credentials/topicCredentials";
+export { RawTokenCredential } from "./credentials/rawTokenCredential";
```

Comment:
It is not clear to me why either of  `TopicCredentials` or `RawTokenCredential` need to be exported....